.circleci/base_config.yml

```diff
@@ -9,7 +9,7 @@ parameters:
     default: ""
   build-docker-image:
     type: string
-    default: "arangodb/build-alpine:3.18-clang16.0-gcc12.2-openssl3.1.4-26c4da3c8"
+    default: "neunhoef/build-ubuntu:23.10-clang16.0.6-gcc13.2.0-openssl3.1.4-6dfb5b28711"
```

Comment:
So far `neunhoef`, since I am not allowed to push to `arangodb`. Can be changed later easily.